client/blocks/reader-subscription-list-item/docs/example.jsx

```diff
@@ -29,9 +28,12 @@ export default class ReaderSubscriptionListItemExample extends PureComponent {
 	render() {
 		return (
 			<Card>
-				{ map( sites, site => (
-					<ConnectedReaderSubscriptionListItem key={ site.feedId || site.siteId } { ...site } />
-				) ) }
+				{ map(
```

Comment:
its not random, but its complicated.  made a change to calypso-prettier since the time we did most of our prettification.